test.py

```diff
@@ -0,0 +1,32 @@
+"""Import modules required to run the Jupyter notebook."""
```

Comment:
As this test should be related to WSIReader. Please add a test in `https://github.com/TissueImageAnalytics/tiatoolbox/blob/develop/tests/test_wsireader.py`

Reply:
Please check the tests in this file which will give you an idea on how to write these tests.